pan3d-js/vite.config.js

```diff
@@ -15,7 +15,7 @@ export default {
         },
       },
     },
-    outDir: "../pan3d/ui/pan3d_components/serve",
+    outDir: "../pan3d/ui/pan3d_components/module/serve",
```

Comment:
don't forget that if you remove `pan3d_components` you will have to fix many location